bindings/pyroot/pythonizations/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -76,7 +76,7 @@ def RDataFrameAsNumpy(df, columns=None, exclude=None):
     for column in columns:
         cpp_reference = result_ptrs[column].GetValue()
         if hasattr(cpp_reference, "__array_interface__"):
-            tmp = numpy.array(cpp_reference) # This adopts the memory of the C++ object.
+            tmp = numpy.asarray(cpp_reference) # This adopts the memory of the C++ object.
```

Comment:
Looks good @stwunsch ! So only `asarray` adopts now and `array` copies?

Reply:
I've checked the adoption when we put this into legacy pyroot, but we haven't had a test for it. For some reason it seems that the `numpy.array` constructor does not adopt anymore. I'm pretty sure it did before! However, `np.asarray` does and we check this now.